samples/client/petstore/python-asyncio/petstore_api/configuration.py

```diff
@@ -466,3 +480,17 @@ def get_host_from_settings(self, index, variables=None):
             url = url.replace("{" + variable_name + "}", used_value)
 
         return url
+
+    @property
+    def host(self):
+        """Return generated host."""
+        if self.server_index is None:
+            return self._base_path
```

Comment:
How about removing this if block because it already exists in get_host_from_settings?